impl/client/src/main/javascript/web/pentaho/type/filter/_core/not.js

```diff
@@ -113,12 +113,14 @@ define([
 
       type: /** @lends pentaho.type.filter.Not.Type# */{
         id: "pentaho/type/filter/not",
+        alias: "not",
 
         styleClass: "pentaho-type-filter-not",
 
         props: [
           {
             name: "operand",
+            nameAlias: "arg",
```

Comment:
I still find it confusing to have a name and a nameAlias that don't share a radical word.